### DISCUSSION TITLE
ScriptReference/Transform-hasChanged

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#532] [Notes]
# ScriptReference/Transform-hasChanged

https://docs.unity3d.com/ScriptReference/Transform-hasChanged.html

**Comment by FreyaHolmer** [comment]
Unity will never set `hasChanged` to false, it only ever sets it to true. In other words, if you don't manually set it to false, this will always return true regardless of whether or not the transform has changed
